server/datastore/mysql/vpp.go

```diff
@@ -579,29 +579,36 @@ func (ds *Datastore) GetTitleInfoFromVPPAppsTeamsID(ctx context.Context, vppApps
 	return &info, nil
 }
 
-func (ds *Datastore) GetVPPAppMetadataByAdamIDAndPlatform(ctx context.Context, adamID string, platform fleet.AppleDevicePlatform) (*fleet.VPPApp, error) {
+func (ds *Datastore) GetVPPAppMetadataByAdamIDPlatformTeamID(ctx context.Context, adamID string, platform fleet.AppleDevicePlatform, teamID *uint) (*fleet.VPPApp, error) {
 	stmt := `
 	SELECT va.adam_id,
 	 va.bundle_identifier,
 	 va.icon_url,
 	 va.name,
+	 va.platform,
 	 va.title_id,
 	 va.platform,
 	 va.created_at, 
 	 va.updated_at,
 	 vat.id
```

Comment:
Since we're now scoping this to a team, we should populate self-service in here to avoid footguns later.